--- xdelta-1.1.4/Makefile.am.orig	2007-03-15 22:30:08.000000000 +0000
+++ xdelta-1.1.4/Makefile.am	2007-03-15 22:30:33.000000000 +0000
@@ -16,10 +16,10 @@
 
 lib_LTLIBRARIES = libxdelta.la
 
-libxdelta_la_SOURCES = xdelta.c xdapply.c $(SER_SOURCES)
+libxdelta_la_SOURCES = xdelta.c xdapply.c xd_edsio.c xd_edsio.h
 libxdelta_la_LIBADD  = $(GLIB_LIBS)
 
-EXTRA_DIST = xd.ser $(SER_OUT) xdelta.magic xdelta.prj xdelta.m4	\
+EXTRA_DIST = xd.ser xdelta.magic xdelta.prj xdelta.m4	\
 	autogen.sh stamp-ser xdrsync.c
 
 SUBDIRS = libedsio . test doc djgpp
@@ -38,13 +38,4 @@
 	$(top_srcdir)/libedsio/edsio-comp xd.ser
 	touch stamp-ser
 
-SER_OUT = xd_edsio.h xd_edsio.c
 
-$(SER_OUT): stamp-ser
-
-SER_SOURCES   = xd_edsio.c
-BUILT_SOURCES = $(SER_SOURCES)
-
-#
-#
-#
